src/UI/styles/autofill-tooltip-styles.css

```diff
@@ -43,6 +41,8 @@
     font-size: 13px;
     line-height: 14px;
     width: 315px;
+    max-height: 290px;
+    overflow-y: auto;
```

Comment:
@alistairjcbrown Watch out that we're now having a max height for the `.tooltip--data`. I'm unsure how we should handle that with your current work on in-context signup. This also includes the "Manage …" button, so we may want to revise the designs (or switch off the Manage … button when in-context is showing). I'm merging this in now.